sdk/spring-sdk/src/test/scala/kalix/springsdk/impl/ActionDescriptorFactorySpec.scala

```diff
@@ -437,6 +441,46 @@ class ActionDescriptorFactorySpec extends AnyWordSpec with ComponentDescriptorSu
         descriptorFor[ActionWithMethodLevelAclAndSubscription]
       }
     }
+
+    "generate mappings for service to service publishing " in {
+      assertDescriptor[EventStreamPublishingAction] { desc =>
+        val serviceOptions = findKalixServiceOptions(desc)
+
+        val eventingOut = serviceOptions.getEventing.getOut
+        eventingOut.getDirect.getEventStreamId shouldBe "employee_events"
+
+        val methodDescriptor = findMethodByName(desc, "KalixSyntheticMethodOnESEmployee")
+        methodDescriptor.isServerStreaming shouldBe false
+        methodDescriptor.isClientStreaming shouldBe false
+
+        val eventingIn = serviceOptions.getEventing.getIn
+        val entityType = eventingIn.getEventSourcedEntity
+        entityType shouldBe "employee"
+
+        val methodOne = desc.commandHandlers("KalixSyntheticMethodOnESEmployee")
+        methodOne.requestMessageDescriptor.getFullName shouldBe JavaPbAny.getDescriptor.getFullName
+
+      }
+    }
+
+    "generate mappings for service to service subscription " in {
+      assertDescriptor[EventStreamSubscriptionAction] { desc =>
+        val serviceOptions = findKalixServiceOptions(desc)
+
+        val eventingIn = serviceOptions.getEventing.getIn
+        val eventingInDirect = eventingIn.getDirect
+        eventingInDirect.getService shouldBe "employee_service"
+        eventingInDirect.getEventStreamId shouldBe "employee_events"
+
+        // we don't set the property so the proxy won't ignore. Ignore is only internal to the SDK
+        eventingIn.getIgnore shouldBe false
+        eventingIn.getIgnoreUnknown shouldBe false
+
+        val methodDescriptor = findMethodByName(desc, "KalixSyntheticMethodOnESEmployee_events")
+        methodDescriptor.isServerStreaming shouldBe false
+        methodDescriptor.isClientStreaming shouldBe false
```

Comment:
This was missing and we were not detecting that a gPRC method was not being generated. 

Note the name of the methods. Ends with `Employee_events'. Not very java style, but this name comes from the name of the stream. Chosen by the use, so not under our control. We could try to remove some symbols, but I don't think it's worth. That's hidden from users.

Reply:
Not doing gives room to hyphens. Just added https://github.com/lightbend/kalix/issues/7911